lib/percy/capybara/client/snapshots.rb

```diff
@@ -46,7 +44,8 @@ def snapshot(page, options = {})
               return
             end
 
-            snapshot['data']['relationships']['missing-resources']['data'].each do |missing_resource|
+            snapshot['data']['relationships']['missing-resources']['data']
```

Comment:
Instead of line-breaking this on the `.each`, will you change this to set this thing to a variable and then just iterate over it on the next line?